jwql/website/apps/jwql/urls.py

```diff
@@ -39,18 +39,33 @@
 """
 
 from django.urls import path
+from django.urls import re_path
 
+from . import api_views
 from . import views
 
 app_name = 'jwql'
+
+api_instruments = 'nircam|NIRCam|niriss|NIRISS|nirspec|NIRSpec|miri|MIRI|fgs|FGS'
+
 urlpatterns = [
     path('', views.home, name='home'),
     path('about/', views.about, name='about'),
     path('dashboard/', views.dashboard, name='dashboard'),
-    path('<str:inst>/', views.instrument, name='instrument'),
-    path('<str:inst>/archive/', views.archived_proposals, name='archive'),
-    path('<str:inst>/unlooked/', views.unlooked_images, name='unlooked'),
-    path('<str:inst>/<str:file_root>/', views.view_image, name='view_image'),
-    path('<str:inst>/<str:file>/hdr/', views.view_header, name='view_header'),
-    path('<str:inst>/archive/<str:proposal>', views.archive_thumbnails, name='archive_thumb'),
+    path('jwql/<str:inst>/', views.instrument, name='instrument'),
+    path('jwql/<str:inst>/archive/', views.archived_proposals, name='archive'),
+    path('jwql/<str:inst>/unlooked/', views.unlooked_images, name='unlooked'),
+    path('jwql/<str:inst>/<str:file_root>/', views.view_image, name='view_image'),
+    path('jwql/<str:inst>/<str:file>/hdr/', views.view_header, name='view_header'),
+    path('jwql/<str:inst>/archive/<str:proposal>', views.archive_thumbnails, name='archive_thumb'),
```

Comment:
This is probably a question for another issue/PR, but do you think we should keep the "jwql" at the front of these URLs? It feels a little redundant, assuming the domain is ultimately going to be jwql.stsci.edu or something.

Reply:
@laurenmarietta I agree that we should try to remove the "jwql", but yes let's tackle that in a separate PR.  It might be tricky.  I tried going down that road as part of this PR and ran into some issues.